docs/bugs.json

```diff
@@ -1,5 +1,17 @@
 [
     {
+        "name": "ConstantOptimizerSubtraction",
+        "short": "In some situations, the optimizer replaces certain numbers in the code with routines that compute different numbers.",
```

Comment:
This doesn't seem to follow the others below, those have `summary` and `description`, this one has `short` and `long`.